litestream/litestream.template.yaml

```diff
@@ -0,0 +1,12 @@
+dbs:
+  - path: /app/data/sqlite/core.db
+    replicas:
+      - type: s3
+        endpoint: #@ data.values.AR_IO_SQLITE_BACKUP_S3_BUCKET_URL
+        name: ar_io_node_core.backup.db
+        bucket: #@ data.values.AR_IO_SQLITE_BACKUP_S3_BUCKET_NAME
+        region: #@ data.values.AR_IO_SQLITE_BACKUP_S3_BUCKET_REGION
+        forcePathStyle: true
+        sync-interval: 1s
+        access-key-id: #@ data.values.AR_IO_SQLITE_BACKUP_S3_BUCKET_ACCESS_KEY
```

Comment:
I'd ideally like the ability to use the EC2 instance role here if no explicit keys are used. That probably the default if the values aren't set explicitly. Putting a conditional around this part of the template based on whether `data.values.AR_IO_SQLITE_BACKUP_S3_BUCKET_ACCESS_KEY` is set should accomplish that.

Reply:
I also wonder if these can be set in such a way that they'd apply to multiple replicas when we add them.

Reply:
litestream will use the instance creds by default - you do not need to set these

Reply:
Nice. I think we do still want support for setting them though. There are probably some users who will run on their own hardware but still backup to S3.

Reply:
The new conditional looks good. I think we should still probably do this though:
> I also wonder if these can be set in such a way that they'd apply to multiple replicas when we add them.
In other words, we should make those global settings.

Reply:
good idea, will implement

Reply:
done and I verified that it's still working with these keys in top-level scope